ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/sorter/GasPricePendingTransactionsTest.java

```diff
@@ -25,7 +25,7 @@
 public class GasPricePendingTransactionsTest extends AbstractPendingTransactionsTestBase {
 
   @Override
-  AbstractPendingTransactionsSorter getSorter(
+  PendingTransactions getSorter(
```

Comment:
rename to getPendingTransaction?

Reply:
thanks for suggesting the renaming, it make sense and I have applied them all